Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook

```diff
@@ -293,7 +334,96 @@
           }
         ]
       },
+      "conditionalVisibility": {
+        "parameterName": "tabName",
+        "comparison": "isEqualTo",
+        "value": "limitsAndUsage"
+      },
       "name": "groupLimitsAndUsage"
+    },
+	{
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": ""
+            },
+            "name": "infoUnusedMetrics"
+          },
+		  {
+			"type": 9,
+			"content": {
+			  "version": "KqlParameterItem/1.0",
+			  "parameters": [
+				{
+				  "id": "05607eff-fcbc-40b8-8eda-e36c4b90a6f7",
+				  "version": "KqlParameterItem/1.0",
+				  "name": "notUsedInDays",
+				  "label": "Not Used In Days",
+				  "type": 2,
+				  "isRequired": true,
+				  "query": "{\"version\":\"1.0.0\",\"content\":\"[\\\"0d\\\", \\\"30d\\\", \\\"60d\\\", \\\"90d\\\"]\",\"transformers\":null}",
+				  "typeSettings": {
+					"additionalResourceOptions": [],
+					"showDefault": false
+				  },
+				  "timeContext": {
+					"durationMs": 86400000
+				  },
+				  "queryType": 8,
+				  "value": "30d"
+				}
+			  ],
+			  "style": "pills",
+			  "queryType": 8
+			},
+			"name": "parameters - 2"
+		  },
+		  {
+			"type": 3,
+			"content": {
+			  "version": "KqlItem/1.0",
+			  "query": "let daysLimit = tolong(substring(\"{notUsedInDays}\", 0, strlen(\"{notUsedInDays}\") - 1));\nAMWMetricsUsageDetails\n| where DaysSinceMetricQueried >= daysLimit\n| summarize [\"Event Count\"] = avg(IncomingEventsCount), [\"Daily Timeseries\"] = avg(DailyTimeseriesCount), [\"Last Queried (days ago)\"] = min(DaysSinceMetricQueried) by [\"Azure Monitor Workspace\"] = _ResourceId, MetricNamespace, MetricName, DimensionsList\n| summarize [\"Event Count\"] = avg([\"Event Count\"]), [\"Daily Timeseries\"] = sum([\"Daily Timeseries\"]), [\"Last Queried (days ago)\"] = min([\"Last Queried (days ago)\"]) by [\"Azure Monitor Workspace\"], MetricNamespace, MetricName",

```

Comment:
fwiw, the substring thing was a bit of a hack to not have to have separate values/labels - IMO nonblocking and your call on if / whether to switch over at this point, but figured I'd mention

Reply:
I took a quick look at it and got the dropdown kind of working without the hack, but it still worked better with the hack so punting this for now

Reply:
Sounds good!